Cookbook generation error using Sphinx from PyPI

The current cmake configuration does not find Sphinx installed from PyPI,
as sphinx-build resides in /usr/local/bin

```
ubuntu@1404:~/shogun/build$ ninja cookbook
[4/5] Generating examples from meta-language
Generating LALR tables
Generating LALR tables
Generating LALR tables
Generating LALR tables
Generating LALR tables
Generating LALR tables
Generating LALR tables
Generating LALR tables
Generating LALR tables
[5/5] Cookbook Sphinx building HTML
FAILED: cd /home/ubuntu/shogun/build/doc/cookbook && /usr/bin/sphinx-build -q -b html -c /home/ubuntu/shogun/build/doc/cookbook/_build -d /home/ubuntu/shogun/build/doc/cookbook/_doctrees -D generated_examples_path="/home/ubuntu/shogun/build/examples/meta/" /home/ubuntu/shogun/doc/cookbook/source /home/ubuntu/shogun/build/doc/cookbook/html
/bin/sh: 1: /usr/bin/sphinx-build: not found
ninja: build stopped: subcommand failed.
```

--- a/cmake/FindSphinx.cmake
+++ b/cmake/FindSphinx.cmake
@@ -13,6 +13,7 @@
 
 find_program(SPHINX_EXECUTABLE
              NAMES sphinx-build sphinx-build2
+             PATH /usr/local/bin
              DOC "Path to sphinx-build executable")
 
 # Handle REQUIRED and QUIET arguments

